Jenkinsfile

```diff
@@ -43,6 +43,64 @@ void createDeploymentStatus (String suffix, String status, String targetURL) {
 }
 
 
+// Functional test script
+// Can be limited by adding smokeTest name
+def functionalTest (String STAGE_NAME, String BASE_URL, String ENV_SUFFIX, String smokeTest='false') {
```

Comment:
I suggest renaming `smokeTest` to `testSet` so it's more clear that the behavior is to run the supplied set of test specs.  The default could be 'all'